layouts/shortcodes/table.html

```diff
@@ -0,0 +1,6 @@
+{{ $htmlTable := .Inner | markdownify }}
```

Comment:
Why this over a garden variety markdown table?

Reply:
Just checked, this is actually needed